recipes/libmemcached/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set series = "1.0" %}
+{% set version = "1.0.18" %}
+{% set pkg = "libmemcached" %}
+
+package:
+  name: {{ pkg }}
+  version: {{ version }}
+
+source:
+  fn: {{ pkg }}-{{ version }}.tar.gz
+  url: https://launchpad.net/{{ pkg }}/{{ series }}/{{ version }}/+download/{{ pkg }}-{{ version }}.tar.gz
+  sha256: e22c0bb032fde08f53de9ffbc5a128233041d9f33b5de022c0978a2149885f82
+  patches:
+    # thanks to http://stackoverflow.com/questions/27004144/how-can-i-install-libmemcached-for-mac-os-x-yosemite-10-10-in-order-to-install-t
+    - osx.configure.patch  # [osx]
+    - osx.memflush.patch  # [osx]
```

Comment:
Hm... this version has been the most recent for some time now. Not sure the right approach to that... this is the version that has been out for some time. Can do some more digging...